service/service.go

```diff
@@ -200,16 +193,31 @@ func New(config Config) (*Service, error) {
 		}
 	}
 
+	var newInformer *informer.Informer
+	{
+		informerConfig := informer.DefaultConfig()
+
+		informerConfig.Watcher = newK8sClient.CoreV1().Services("")
+
+		informerConfig.RateWait = config.Viper.GetDuration(config.Flag.Service.Controller.ResyncPeriod)
+		informerConfig.ResyncPeriod = config.Viper.GetDuration(config.Flag.Service.Controller.ResyncPeriod)
```

Comment:
I recommend using defaults unless there is a particular reason not to do so. They are sane and we can update them globally.

Why using flags for that? Should those values be different between installations?

Reply:
Yeah, legit. I need to sort out some general flag foo anyway, so I'll do this in a followup.